test/oscar/compiler/arithFunctionChaining.oscar

```diff
@@ -0,0 +1,15 @@
+def f1(num: int) => int = {
+  return int * 2;
```

Comment:
Actually none of the test files end in newline. For our language's style guideline, do we want to have a trailing newline or no?